src/model_multibert.py

```diff
@@ -0,0 +1,103 @@
+import torch
```

Comment:
`model_multibert.py` was merged with `model.py` - please pull from the latest commit